bene_media.module

```diff
@@ -429,3 +432,67 @@ function bene_media_menu_links_discovered_alter(&$links) {
     $links['entity.media.collection']['route_name'] = 'view.bene_media.media_page_list';
   }
 }
+
+/**
+ * Adds the media_tags field to a given bundle.
+ *
+ * @param string $bundle
+ *   The media bundle ID.
+ */
+function _bene_media_update_add_tags($bundle) {
```

Comment:
I kind of want this to be in an include file instead of right here in the .module file. Not a deal breaker, but what do you think @mortenson

Reply:
@mariacha That sounds OK, this only needs to be used at install time. Do you want it in an include file for organizational purposes only or to be included only in `.install` files?

Reply:
Mostly the latter -- one less function for PHP to parse at run time during non-install queries. Something like an `install.inc`

Reply:
@mariacha Alright, I moved it and added an include to all of that function's uses.